client/lib/domains/types.ts

```diff
@@ -101,6 +101,7 @@ export type ResponseDomain = {
 	registrar: string;
 	registrationDate: string;
 	renewableUntil: string;
+	registryExpiryDate: string | null;
```

Comment:
I think this could be only `string` (`assembler.js` force an empty string in case there is no value from API).

```suggestion
	registryExpiryDate: string;
```

Reply:
That's a good point! I'll update this 👍